execution_workflows/LABRAT/modules/labrat_calculatepsi.nf

```diff
@@ -8,20 +8,26 @@ process LABRAT_CALCULATEPSI {
         publishDir "${params.outdir}/labrat", mode: params.publish_dir_mode
         container "quay.io/biocontainers/labrat:0.2.2--pyhdfd78af_0"
 
+        cpus 3
+        memory '16 GB'
+
```

Comment:
Hm yeah I probably set this because I believed the task could not profit from more. 
This was on some real data, so should've worked with that. 
I was not aware that the memory and cpu are treated as actual values and not as limits as in snakemake.
Of course I can remove the statement, shall I do this?